.github/dockerbuild.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: image-builder
+
+on:
+  push:
+    branches: ['*']
+  pull_request:
+    branches: ['*']
+
+jobs:
+  docker-build:
+    name: docker-build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.16
```

Comment:
This is now defined in 2 places, here and in the make file.

Can we get the go-version from an env var, and set it from somewhere within the project so both make and the workflow with use it? It will also be helpful to allow us to override the go-version when running locally or dispatching manually

Reply:
I don't think it is good to set the go version using the environment variable in go.mod and Dockerfile.

Reply:
I think you misunderstand me. 
My ask is to read it from one of these sources and use it here, you are already checking-out the entire source repo so you can read it from the go.mod file

Reply:
Ahh... Yes, I misunderstood your suggestion. I'll update the PR with the necessary changes.